modular_zubbers/code/modules/power/powerator.dm

```diff
@@ -180,7 +180,7 @@
 	if(!attached_cable.avail(current_power))
 		if(!attached_cable.newavail())
 			return
-		current_power = attached_cable.newavail()
+		current_power = min(max_power, attached_cable.newavail())
 	attached_cable.add_delayedload(current_power)
```

Comment:
```suggestion
	attached_cable.add_load(current_power)
```
Similarly, machines are supposed to use `add_load()` instead of `add_delayedload()`